newsfragments/197.feature.md

```diff
@@ -0,0 +1 @@
+The CLI now compiles every contract in a module, not just the first one.
```

Comment:
Maybe worth hinting about the compiled output with the example from the PR description.

```
output
|-- Bar
|   |-- Bar.bin
|   |-- Bar_abi.json
|   `-- Bar_ir.yul
|-- Foo
|   |-- Foo.bin
|   |-- Foo_abi.json
|   `-- Foo_ir.yul
|-- module.ast
`-- module.tokens
```